clickfile.py

```diff
@@ -1266,16 +1302,38 @@ def run(network, script):
     if os.environ.get('FAUCET_URL') is None:
         os.environ["FAUCET_URL"] = config[network]['faucet_url']
 
-    os.environ["TRACER_URL"] = config[network]['tracer_url']
-    os.environ["SPL_NEON_MINT"] = config[network]['spl_neon_mint']
-    os.environ["NEON_ERC20_WRAPPER_ADDRESS"] = config[network]['neon_erc20wrapper_address']
-    os.environ["NETWORK_ID"] = str(config[network]['network_ids']['neon'])
+    if os.environ.get('TRACER_URL') is None:
+        os.environ["TRACER_URL"] = config[network]['tracer_url']
+
+    if os.environ.get('SPL_NEON_MINT') is None:
```

Comment:
why ado you need a possibility to replace all env variables from  local environment instead of envs.json ?  we have there all networks,  it is more transparent to use urls and addresses as it used for another tests

Reply:
I get all params from envs.json and set up everything as envs for k6, it is how k6 gets needed info.

Reply:
Yes, i see
but why do you need conditions like this
if os.environ.get('SPL_NEON_MINT') is None:

Reply:
all unused evns are removed